src/frontend/src/flows/register/passkey.ts

```diff
@@ -10,26 +10,39 @@ import {
   isWebAuthnCancel,
   webAuthnErrorCopy,
 } from "$src/utils/webAuthnErrorUtils";
-import { html, TemplateResult } from "lit-html";
+import { html, nothing, TemplateResult } from "lit-html";
 import { registerStepper } from "./stepper";
 
 import copyJson from "./passkey.json";
 
 /* Anchor construction component (for creating WebAuthn credentials) */
 
 const savePasskeyTemplate = ({
-  construct,
+  constructPasskey,
+  constructPin,
   i18n,
   cancel,
   scrollToTop = false,
 }: {
-  construct: () => void;
+  constructPasskey: () => void;
+  constructPin?: () => void;
   i18n: I18n;
   cancel: () => void;
   /* put the page into view */
   scrollToTop?: boolean;
 }): TemplateResult => {
   const copy = i18n.i18n(copyJson);
+  const constructPinButton = constructPin
+    ? html`
+        <button
+          @click=${() => constructPasskey()}
+          data-action="construct-pin-identity"
+          class="c-button c-button--secondary"
```

Comment:
if this button is `secondary` shouldn't it find place after the primary UI wise?

Reply:
It is placed second, this is just preparing the element to then slot into the other element below.